src/Categories/Multi/Category/Cartesian/Representable.agda

```diff
@@ -0,0 +1,174 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Multi.Category.Cartesian.Representable where
+
+open import Data.List hiding ([_])
+open import Data.List.Membership.Propositional
+open import Data.List.Membership.Propositional.Properties
+open import Data.List.Relation.Unary.Any
+import Data.List.Relation.Unary.Any.Properties as AnyP
+open import Data.Product
+open import Data.Sum
+open import Data.Wrap
+open import Function.Base as F using (case_return_of_)
+open import Level
+open import Relation.Binary.PropositionalEquality as ≡ using (_≡_)
+
+open import Categories.Category
+open import Categories.Category.Cartesian
+import Categories.Category.Cartesian.Properties as ×P
+open import Categories.Multi.Category.Cartesian
+
+private
+  variable
+    o ℓ e : Level
+
+module Extras {a} {A : Set a} where
+  -- TODO: contribute to stdlib
```

Comment:
I think some of these have landed in stdlib now. But yes, do push these to there, they can be eliminated from here 'later'.

Reply:
I don't spot any of them.

Reply:
Ah indeed. I was fooled because they do exist for `Fin`, but not for `List`.